### ISSUE TITLE
Bootstrap dropdown transform causes overflow

### ISSUE THREAD

**Issue by jchapelle** [ng-bootstrap/ng-bootstrap#3640]
### Bug description:
I'm using a dropdown inside the last cell of a table. The dropdow menu is large. For that reason, I'm using the bottom-right placement of the menu. The menu is properly positionned but the css transform property seems to introduce some strange overflow in the width of the window.
![dropdown-issue](https://user-images.githubusercontent.com/6162013/76260376-f4629680-6257-11ea-8595-4de1571bd5b0.png)

When applying the exact same value of the transform css property to the top and left css properties, it seems to work fine ... 
![dropdown-issue-2](https://user-images.githubusercontent.com/6162013/76260687-aac67b80-6258-11ea-81d0-b2fc5392afa8.png)

### Link to minimally-working StackBlitz that reproduces the issue:
You have to click at least 2 times for the bug to occurs
Also, the bug depends of the size of your window. Maybe you'll have to add some cell in the table row. You'll have to get the dropdow at the very right of your window (cfr screenshiot) for the bug to occur
https://stackblitz.com/edit/angular-74szwx

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 9

ng-bootstrap: 6

Bootstrap: 4.4.1

**Comment by benouat** [comment]
Closed in #3644

**Comment by maxokorokov** [comment]
This fix broke popover in `6.0.1`, so this is reverted in `6.0.2`

**Comment by jchapelle** [comment]
In addition, the fix wasn't really working fine. A blink effect on horizontal scrollbar was introduced as well

**Comment by fbasso** [comment]
I tested it with pure bootstrap stuff. It seems that it's the same kind of issue. By default (with Bootstrap), the tooltip is set on the body. When I set it explicitly inside the table, the scrollbars appear:

![image](https://user-images.githubusercontent.com/1152930/77519896-99988400-6e80-11ea-9413-6d4833b1fd22.png)

To solve the issue, I think the best way is to set it explicitly on the body with container="body": https://ng-bootstrap.github.io/#/components/tooltip/examples#container
